src/main/resources/roleconfig/role_common.json

```diff
@@ -688,6 +688,42 @@
       }
     ]
   },
+  {
+    "name": "specific-access-judiciary",
```

Comment:
no need to use another role config, use existing specific-access-judiciary and add jurisdiction in attributes. Don't worry about caseType